src/pkg/selectors/selectors.go

```diff
@@ -190,6 +177,36 @@ func (s Selector) PathService() path.ServiceType {
 	return serviceToPathType[s.Service]
 }
 
+type reducer interface {
+	Reduce(ctx context.Context, deets *details.Details) *details.Details
+}
```

Comment:
I believe this is already defined towards the top of the file as an exported interface

Reply:
haha, right you are, thanks for the reminder.  I kicked one of those out.